hr_employee_firstname/models/hr_employee.py

```diff
@@ -34,10 +33,11 @@ def split_name(self, name):
         clean_name = u" ".join(name.split(None)) if name else name
         return self.env['res.partner']._get_inverse_name(clean_name)
 
-    @api.cr_context
-    def _auto_init(self, cr, context=None):
-        super(HrEmployee, self)._auto_init(cr, context=context)
-        self._update_employee_names(cr, SUPERUSER_ID, context=context)
+    @api.model_cr_context
+    def _auto_init(self):
```

Comment:
I think this should be done right here. Migration means sometimes a refactorization, and something that is not correct at all (this is not multi-DB aware) should be cut from the beginning.